api/projects.ts

```diff
@@ -1,9 +1,27 @@
 import { axios } from "./axios";
-import type { Project } from "./projects.types";
+import { Project, ProjectStatus } from "./projects.types";
 
 const ENDPOINT = "/project";
 
-export async function getProjects() {
+export async function getProjects(): Promise<Project[]> {
   const { data } = await axios.get<Project[]>(ENDPOINT);
-  return data;
+
+  const modifiedData = data.map((project, index) => {
+    let status: ProjectStatus = ProjectStatus.stable; // Default value
+
+    if (index === 0) {
+      status = ProjectStatus.critical;
+    } else if (index === 1) {
+      status = ProjectStatus.warning;
+    } else if (index === 2) {
+      status = ProjectStatus.stable;
+    }
+
+    return {
+      ...project,
+      status,
+    };
```

Comment:
The problem with this code is that it's not dynamic. Always the first project is `critical`, the second `warning`, and the third `stable`. But this is just the current data set. The API might return it in a different order. And actually there are usually more users/organizations that would use this app and have different projects with different status.

The goal of this task was to map the existing `status` field with values `error`, `warning` and `info` to the display values `Critical`, `Warning`, and `Info`. 

![CleanShot 2024-08-12 at 11 22 27](https://github.com/user-attachments/assets/da3980cf-9369-4937-8696-a4ab10f786ec)


This can be done with a map object in the component like this (just a quick sketch of example code)

```
const statusToDisplayText = {
  [ProjectStatus.error]: 'Critical',
  [ProjectStatus.warning]: 'Warning',
  [ProjectStatus.info]: 'Stable',
}

<div>
  {statusToDisplayText[project.status]}
</div>
```

The `ProjectStatus` would have to be changed as well to match the actual data structure of the API requests